core-bundle/src/Twig/Extension/ContaoExtension.php

```diff
@@ -293,6 +293,11 @@ public function getFilters(): array
                 [SanitizerRuntime::class, 'sanitizeHtml'],
                 ['is_safe' => ['html']],
             ),
+            new TwigFilter(
+                'extract_styles_for_csp',
+                [CspRuntime::class, 'extractStyleAttributes'],
+                ['is_safe' => ['html']],
```

Comment:
```suggestion
                ['preserves_safety' => ['html']],
```
Otherwise this filter would be an implicit `|raw` I think. /cc @m-vo ?